src/utils/formatTime.js

```diff
@@ -40,4 +40,13 @@ function formatCountdown (seconds) {
   }
 }
 
-export { formatDate, formatTime, formatToUTC, formatCountdown }
+function calculateTime (session) {
```

Comment:
This function could really use a couple of comments and/or a more descriptive name, it's hard to know at a glance what it's doing. What time is it calculating?

Reply:
adding comments. But it returns a countdown. Calculates either time remaining in the session or time before the session starts

Reply:
maybe renaming it to calculateSessionCountdown would be a little more descriptive?

Reply:
hmmm yes and it sometimes is time remaining in session. Would that name make sense then? (English isn't my first language)